docs/template.html

```diff
@@ -52,13 +52,13 @@
     class="silver bg-raisin-black essential-pragmata-pro f5 lh-copy nested-list-reset"
   >
     {{ if ne .Filename "index.html"}}
-      <header>
-        <a class="flex no-underline f2-ns f3 white calibre" href="/index.html"
+      <header class="cf pa3">
+        <a class="flex no-underline f2-ns f3 white calibre fl" href="/index.html"
           >please<span class="violet">.build</span></a
         >
         <button
                 id="menu-button"
-                class="white bg-transparent b--none f3 pointer dn-l"
+                class="white bg-transparent b--none f3 pointer dn-l fr"
```

Comment:
I would use flex here since it's easier to reason about and has less quirks than floats.

`flex justify-between` to the header class should be enough to get the same result